docs/chapters/licensing/index.rst

```diff
@@ -0,0 +1,23 @@
+Licensing
+=========
+
+Each source code file needs to mention which license applies and who owns
+the copyright in the file header. The purpose of this is to make it
+absolutely clear, for both humans and tooling, which license applies to
+that particular file even when the file is moved to another project. In
+order to achieve this, it has been decided that the header should include:
```

Comment:
I still don't like the way this reads. Could you remove the "it has been decided"?